prost-build/src/ast.rs

```diff
@@ -134,7 +134,7 @@ pub struct Service {
     /// The package name as it appears in the .proto file.
     pub package: String,
     /// The service comments.
-    pub comments: Comments,
+    pub comments: Option<Comments>,
```

Comment:
instead of making this breaking change can we instead make it so comments has a `Comments::default()`?

Reply:
Looking at prost-build/src/lib.rs:154, the ast module isn't public, so I assumed this was an implementation detail of `prost-build`.
```rust
mod ast
```
Did I misunderstand that somehow?

Reply:
oh, nevermind.  Missed line 147:
```rust
pub use crate::ast::{Comments, Method, Service};
```

Reply:
Added a derived `Default` to `Comment`.  I went that direction instead of directly defining one since all of the fields of `Comment` are public, so unlikely to change, and the derived default is what I'd expect here.